src/plugins/data_views/server/fetcher/lib/field_capabilities/field_capabilities.ts

```diff
@@ -31,7 +31,7 @@ interface FieldCapabilitiesParams {
  *  @param  {Array}  [indices=[]]  the list of indexes to check
  *  @param  {Array}  [metaFields=[]] the list of internal fields to include
  *  @param  {Object} fieldCapsOptions
- *  @return {Promise<Array<FieldDescriptor>>}
+ *  @return {Promise<{ fields: Array<FieldDescriptor>, indices: Array<string>>}>}
```

Comment:
nit: TSDoc spec doesn't have a type in `@param` and `@returns` blocks definitions. That could simply be removed.